app/mva/fork_test.go

```diff
@@ -298,14 +394,19 @@ func (suite *ForkTestSuite) TestFork() {
 				}
 				suite.tstaking.Delegate(tc.args.acc.GetAddress(), operAddr, s.Int64())
 			}
-			for i, u := range tc.args.unbondings {
-				operAddr, err := sdk.ValAddressFromBech32(suite.sk.GetAllValidators(suite.ctx)[i].OperatorAddress)
-				if err != nil {
-					panic(err)
+			suite.tstaking.TurnBlock(suite.ctx)
+
+			if len(tc.args.unbondings) != 0 {
+				for i, u := range tc.args.unbondings {
+					operAddr, err := sdk.ValAddressFromBech32(suite.sk.GetAllValidators(suite.ctx)[i].OperatorAddress)
+					if err != nil {
+						panic(err)
+					}
+					suite.tstaking.Undelegate(tc.args.acc.GetAddress(), operAddr, u.Int64(), true)
 				}
-				suite.tstaking.Undelegate(tc.args.acc.GetAddress(), operAddr, u.Int64(), true)
+				suite.tstaking.TurnBlockTimeDiff(suite.tstaking.GetUnbondingTime())
```

Comment:
delete this line? this only tests once unbondings complete so we can't test if unbonding is correctly summed up